.github/workflows/check_libxrpl.yml

```diff
@@ -27,7 +27,13 @@ jobs:
       - name: Update libXRPL version requirement
         shell: bash
         run: |
-          ./.github/scripts/update-libxrpl-version ${{ github.event.client_payload.version }}
+          sed -i.bak -E "s|'xrpl/[a-zA-Z0-9\\.\\-]+'|'xrpl/${{ github.event.client_payload.version }}'|g" conanfile.py
```

Comment:
I assume this works on all supported platforms

Reply:
Yes